examples/example02_trip98.py

```diff
@@ -30,14 +30,16 @@
 logging.basicConfig(level=logging.INFO)  # give some output on what is going on.
 
 # Fist we specify the directory where all our files are:
-wdir = "/home/bassler/test"
+wdir = "/media/deerjelen/backup/working/IFJ/temporary" # working dir must exist.
```

Comment:
I suggest to make paths at this point more anonymous. 
What about:
`wdir = /home/user/workspace`
and
`patient_dir = /home/user/data/yoda`
and
`trip_path=/home/user/usr/trip98`

A comment should be added like this:
# Please adjust these paths accoring to location of the patient data (CT and contouring) and TRiP98 installation.